### DISCUSSION TITLE
File Missing in Pipeline but showing in PR

### DISCUSSION THREAD

**Discussion by Ronak-Toshniwal** [scolladon/sfdx-git-delta#430] [Q&A]
Hi

I have a pipe made on bitbucket 
where i am getting files to be validated/deployed to prod. 
Now i have created a new field and added the xml file in correct directory in respective branch.
Then i have created a new PR and it is getting all the correct files which need to be validated against PROD.
my pipeline runs on PR creation and sfdx-git-delta plugin is used to create delta changes package.xml in pipe and only those things are VALIDATED.
I am getting a the new field and all other files in PR but new field is NOT SHOWING in PIPE all other files are available

can anyone help?

**Comment by scolladon** [comment]
Can you show us the pipeline used in this scenario ?
